src/setup/empireNames.ts

```diff
@@ -0,0 +1,157 @@
+
+export const empireNames: string[] = [
+    "Nova Empire",
+    "Eldor Kingdom",
+    "Arcadian Dominion",
+    "Sylvan Dynasty",
+    "Aetheria Empire",
+    "Solstice Kingdom",
+    "Aquaterra Dominion",
+    "Frostland Dynasty",
+    "Emberia Empire",
+    "Mythorian Kingdom",
+    "Ironhold Dominion",
+    "Shadowrealm Dynasty",
+    "Verdantia Empire",
+    "Stormguard Kingdom",
+    "Crystalia Dominion",
+    "Phoenix Dynasty",
+    "Dragonspire Empire",
+    "Aurora Kingdom",
+    "Lunaris Dominion",
+    "Titan's Dynasty",
+    "Starhaven Empire",
+    "Dreamland Kingdom",
+    "Sunfire Dominion",
+    "Silvercrest Dynasty",
+    "Everglen Empire",
+    "Mistwood Kingdom",
+    "Echovalley Dominion",
+    "Wyvernguard Dynasty",
+    "Moonshadow Empire",
+    "Astral Kingdom",
+    "Valiant Dominion",
+    "Frostforge Dynasty",
+    "Emberhold Empire",
+    "Celestial Kingdom",
+    "Ironwood Dominion",
+    "Stormhelm Dynasty",
+    "Phoenixfire Empire",
+    "Dragonreach Kingdom",
+    "Arboria Dominion",
+    "Solaris Dynasty",
+    "Moonstone Empire",
+    "Verdant Dominion",
+    "Nightfall Dynasty",
+    "Crystalis Empire",
+    "Aurora Kingdom",
+    "Titan's Dominion",
+    "Starfall Dynasty",
+    "Dreamgate Empire",
+    "Suncrest Kingdom",
+    "Silverwood Dominion"
+];
```

Comment:
something cool would be to split this into prefix-suffix then randomize both - so might get Stormhelm Empire etc